src/internal/gateways/openedge/action-validate-data.ts

```diff
@@ -0,0 +1,32 @@
+import { IErrorReason } from "../";
+import { IDictionary } from "../../lib/util";
+
+export default (data: IDictionary) => {
+  const errors: IErrorReason[] = [];
+  const cardNumber = data["card-number"].replace(/\s+/g, "");
+
+  // The error message here is irrelevant - actual 'invalid_input' error is generated in tokenize.ts.
+  // For type compatibility reason, this code is preserved here.
+  // if (!data["card-number"] && !data["card-track"] && !data["account-number"]) {
+  if (cardNumber.length < 13 || cardNumber.length > 19) {
+    errors.push({
+      code: "invalid_input",
```

Comment:
The error code here should be consistent with the other gateways when validating the card number.

Reply:
Error code is consistent with the API